content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -81,12 +81,13 @@ Hit enter for command prompt
 
 / # wget --spider --timeout=1 nginx
 Connecting to nginx (10.100.0.16:80)
+remote file exists
 / #
 ```
 
-## Limit access to the `nginx` service
+## Limit access to the nginx Service
```

Comment:
I'd leave the backticks here. `nginx` _is_ the name of the Service, right?

Reply:
Fixed.